### PR TITLE
Support load_minigraph command for multi NPU platform

### PR DESCRIPTION
**Pull request by arlakshm**
Signed-off-by: Arvindsrinivasan Lakshmi Narasimhan <arlakshm@microsoft.com>

**- What I did**
Modify the load_minigraph command handler to support multi NPU platforms
 This PR needs the changes from[#4222](https://github.com/Azure/sonic-buildimage/pull/4222) for the minigraph parsing changes

**- How I did it**

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**
No command output is changed
**- New command output (if the output of a command-line utility has changed)**
No command output is changed

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -656,26 +659,36 @@ def load_minigraph():
...
+            command = "{} -H -m -j /etc/sonic/init_cfg.json {} --write-to-db".format(SONIC_CFGGEN_PATH, cfggen_namespace_option)
+        else:
+            command = "{} -H -m --write-to-db {} ".format(SONIC_CFGGEN_PATH,cfggen_namespace_option)
+        run_command(command, display_cmd=True)
+        client.set(config_db.INIT_INDICATOR, 1)
+        if device_type != 'MgmtToRRouter':
+            run_command('{} pfcwd start_default'.format(ns_cmd_prefix), display_cmd=True)
+        if os.path.isfile('/etc/sonic/acl.json'):
+            run_command("acl-loader update full /etc/sonic/acl.json", display_cmd=True)
+        run_command("{} config qos reload".format(ns_cmd_prefix), display_cmd=True)
```
Comment (lguohan) [inline config/main.py:691]: 687-691 do not apply to default namespace in multi-asic environment.

Comment (arlakshm) [reply config/main.py]: Fixed in the latest commit

**Comment by arlakshm** [comment]
retest default please

**Comment by rlhui** [comment]
@arlakshm, there's conflict cherry-picking. please raise a separate PR for 201911 branch.

**Pull request merged**